docs/javascript/v2/release-notes/release-notes.mdx

```diff
@@ -14,6 +14,29 @@ description: Release Notes for 100ms JavaScript SDK
 | @100mslive/hls-player                  | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhls-player.svg)](https://badge.fury.io/js/%40100mslive%2Fhls-player)           |
 | @100mslive/hms-video-react(deprecated) | [![npm version](https://badge.fury.io/js/%40100mslive%2Fhms-video-react.svg)](https://badge.fury.io/js/%40100mslive%2Fhms-video-react) |
 
+## 2023-11-06
+Released: `@100mslive/hms-video-store@0.10.19`, `@100mslive/react-sdk@0.8.19`, `@100mslive/hls-player@0.1.19`, `@100mslive/roomkit-react@0.1.10`
+
+### Added:
+- Roomkit Prebuilt: 
+    - Peers can now select virtual backgrounds to apply instead of getting a random background each time.
+    - Mweb header now indicates the type of audio device selected.
+
+### Fixed:
+- Network indicator tooltip in the participant list would get hidden for the last peer.
+- Roomkit Prebuilt: Preview would show go live CTA even when browser recording was running, resulting in failure to go live.
```

Comment:
We should add an introduction of states in hls, recording. We also had a functionality to pause/resume hls recording using API. @raviteja83 @KaustubhKumar05

Reply:
that will be in the next week's release.